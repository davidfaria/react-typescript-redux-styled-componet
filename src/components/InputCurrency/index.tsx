import * as S from './styles'

const InputCurrency = () => (
  <S.Container>
    <h1>InputCurrency</h1>
  </S.Container>
)

export default InputCurrency
