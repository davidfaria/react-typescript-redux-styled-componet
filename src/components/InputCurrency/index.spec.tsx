import { render, screen } from '@testing-library/react'

import InputCurrency from '.'

describe('<InputCurrency />', () => {
  it('should render the heading', () => {
    const { container } = render(<InputCurrency />)

    expect(
      screen.getByRole('heading', { name: /InputCurrency/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
