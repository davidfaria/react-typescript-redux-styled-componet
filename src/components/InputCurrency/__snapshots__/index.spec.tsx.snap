// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputCurrency /> should render the heading 1`] = `
<div
  class="sc-bdfBwQ"
>
  <h1>
    InputCurrency
  </h1>
</div>
`;
